src/contribute/contribution-guide.md

```diff
@@ -458,8 +458,15 @@ When you are ready, submit a pull request using the [Beam Site GitHub mirror](ht
 
 During review, committers will patch in your PR, generate the static `content/`, and review the changes.
 
-#### Committing website changes
+#### Committing website changes (committers only)
 
 Follow the same committer process as above, but using repository `apache/beam-site` and branch `asf-site`.
 
-In addition, the committer is responsible for doing the final `jekyll build` to generate the static content, so follow the instructions above to install `jekyll`.
+In addition, the committer is responsible for doing the final `bundle exec jekyll build` to generate the static content, so follow the instructions above to install `jekyll`.
+
+This command generates the `content/` directory. The committer should add and commit the content related to the PR. Pay attention to add only the files related to the given pull request since `jekyll` regenerates all the website and sometimes there are unrelated changes e.g. Updated timestamps for old blog posts that should not be added to the website repository among others.
```

Comment:
You are right Davor, much better solution, I am going to fix the timezone to America/Los_Angeles because this is the default one so it won't clash with the timestamps. I will add this to the PR.